packages/flutter/lib/src/material/navigation_drawer.dart

```diff
@@ -354,8 +359,9 @@ class _NavigationDestinationBuilder extends StatelessWidget {
       ],
     );
 
-    return Padding(
+    return Container(
       padding: info.tilePadding,
+      color: backgroundColor ?? navigationDrawerTheme.backgroundColor,
```

Comment:
Should we also add `?? defaults.backgroundColor` after `navigationDrawerTheme.backgroundColor`?

Reply:
it's not necessary because drawer itself will fallback to defaults.backgroundColor after drawerTheme.backgroundColor and navigationDrawerTheme.backgroundColor

Reply:
Got it! Thanks for the explanation!